docs/strategy-customization.md

```diff
@@ -639,6 +639,157 @@ Stoploss values returned from `custom_stoploss` must specify a percentage relati
+    is blocked by `confirm_trade_exit()` method. Warnings can be solved by never blocking stop loss sells by checking `sell_reason` in
+    `confirm_trade_exit()`, or by using `return stoploss_from_open(...) or 1` idiom, which will request to not change stop loss when
+    `current_profit < open_relative_stop`.
+
+### *stoploss_from_absolute()*
+
+In some situations it may be confusing to deal with stops relative to current rate. Instead, you may define a stoploss level using an absolute price.
+
+??? Example "Returning a stoploss using absolute price from the custom stoploss function"
+
+    Say the open price was $100, and `current_price` is $121 (`current_profit` will be `0.21`).
+
+    If we want a stop price at $107 price we can call `stoploss_from_absolute(107, current_rate)` which will return `0.1157024793`.  11.57% below $121 is $107, which is the same as 7% above $100.
+
+    ``` python
+
+    from datetime import datetime
+    from freqtrade.persistence import Trade
+    from freqtrade.strategy import IStrategy, stoploss_from_open
+
+    class AwesomeStrategy(IStrategy):
+
+        # ... populate_* methods
+
+        use_custom_stoploss = True
+
+        def custom_stoploss(self, pair: str, trade: 'Trade', current_time: datetime,
+                            current_rate: float, current_profit: float, **kwargs) -> float:
+
+            # once the profit has risen above 10%, keep the stoploss at 7% above the open price
+            if current_profit > 0.10:
+                return stoploss_from_absolute(trade.open_rate * 1.07, current_rate)
+
+            return 1
+
+    ```
+
+### *@informative()*
+
+``` python
+def informative(timeframe: str, asset: str = '',
+                fmt: Optional[Union[str, Callable[[KwArg(str)], str]]] = None,
+                ffill: bool = True) -> Callable[[PopulateIndicators], PopulateIndicators]:
+    """
+    A decorator for populate_indicators_Nn(self, dataframe, metadata), allowing these functions to
+    define informative indicators.
+
+    Example usage:
+
+        @informative('1h')
+        def populate_indicators_1h(self, dataframe: DataFrame, metadata: dict) -> DataFrame:
+            dataframe['rsi'] = ta.RSI(dataframe, timeperiod=14)
+            return dataframe
+
+    :param timeframe: Informative timeframe. Must always be equal or higher than strategy timeframe.
+    :param asset: Informative asset, for example BTC, BTC/USDT, ETH/BTC. Do not specify to use
+    current pair.
+    :param fmt: Column format (str) or column formatter (callable(name, asset, timeframe)). When not
+    specified, defaults to:
+    * {base}_{column}_{timeframe} if asset is specified and quote currency does match stake
+    currency. 
+    * {base}_{quote}_{column}_{timeframe} if asset is specified and quote currency does not match 
+    stake currency. 
+    * {column}_{timeframe} if asset is not specified.
```

Comment:
I still find the default behaviour of this confusing.

Assume this case: `@informative('1h', asset='BTC/BUSD')` (which also adds an `rsi` column).

The outcome will be:
* `btc_busd_rsi_1h` if my stake-currency is USDT
* `btc_rsi_1h` if my stake-currency is BUSD

This makes it difficult to work with in cases where you wanna switch stake-currency (which you'll probably not think about when first implementing this in a strategy) - while keeping the informative pair consistent.

i know we can define it with the format explicitly - but the default seems inconsistent and difficult to explain as it changes based on the used stake-currency.

While it'll force me to then use `f"btc_busd_rsi_1h"` (which is 5 characters longer than the alternative) - i can rely on this to work even if i switch stake-currency (but would still like to use the same informative pair).

Reply:
> While it'll force me to then use f"btc_busd_rsi_1h" (which is 5 characters longer than the alternative) - i can rely on this to work even if i switch stake-currency (but would still like to use the same informative pair).

That would force you to change strategy source code every time stake currency is changed (most common case is using BTC/STAKE as informative). This is no good...

Maybe a compromise would be `BTC/BUSDT` expanding to `btc_busd_rsi_1h` and `BTC/{stake}` expanding to `btc_rsi_1h`? One more magical rule though. I dont think we can save people from not reading docs.

Reply:
Well we need some kind of a decision. What do you think would be a proper thing to do, hopefully not introducing a need for source code modifications each time stake is changed?

Reply:
i need to change sourcecode now due to the inconsistent "magic" that's happening.

`@informative('1h', asset='BTC/BUSD')` expands to 2 different things depending on something unrelated ( change of `stake_currency`) -  so you force users to change sourcecode when it's not necessary.
It should always expand to `btc_busd_rsi_1h` - then it's whats expect (it's a stable, hardcoded pair, that's not changing when i change stake-currency).

`@informative('1h', asset='BTC/{stake}')` on the other hand is different, and can expand to `btc_rsi_1h` -  it's an explicit user choice of using BTC vs. {stake} - not BTC/USDT or BTC/USDT (which is also an explicit choice - to always compare against that particular pair). 
It'll also not change if i switch from USDT to BUSD (or EUR) - but it'll be always the same (always BTC vs. "my stake currency".

There is a huge difference between the user intend here (what the user desired to do) - one is "always use BTC/USDT, because i explicitly told you so and don't care about BTC/BUSD" - the other is "use BTC/{stake}" - so always use BTC/<my current stake>.

It's therefore not "one more magical rule" - but it's "less" magic - with predictable outcome.